packages/portal/src/components/search/SideFilters.vue

```diff
@@ -15,21 +15,7 @@
         <b-row
           class="border-bottom border-top d-flex justify-content-between align-items-center flex-nowrap"
         >
-          <div
-            v-if="totalResults"
-            class="filters-title"
-            data-qa="total results"
-          >
-            {{ $tc('items.itemCount', totalResultsLocalised, { count: totalResultsLocalised }) }}
-            <div
-              class="visually-hidden"
-              role="status"
-            >
-              {{ $t('searchHasLoaded', [totalResultsLocalised]) }}
-            </div>
```

Comment:
We need this status message for accessibility.

Reply:
Indeed, my apologies, thank you for restoring it (elsewhere).